src/index.js

```diff
@@ -75,6 +76,41 @@ function analyzeManagerClick(event) {
   }
 }
 
+function analyzeCustomerClick(event) {
+  let availableRooms = getRoomsAvailableOnDate();
+  if (event.target.id === 'reservation-search') {
```

Comment:
This is much cleaner than using event.target.classList.contains('#id-goes-here')... I may just have to use it from now on! 💯